RELEASES.md

```diff
@@ -53,6 +53,7 @@ Language
   ```
 - [`extern` functions will now abort by default when panicking.][55982]
   This was previously undefined behaviour.
+- [You may now use `Rc`, `Arc` and `Pin` as method receivers][56805].
```

Comment:
Maybe `Rc<Self>` and so on? That'll make it more clear what this means.

Reply:
Yep, per https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#using-markdown.